dcos/http.py

```diff
@@ -497,6 +497,29 @@ def _get_dcos_auth(auth_scheme, username, password, hostname):
     return DCOSAcsAuth(token)
 
 
+def check_cluster_capability(capability):
+    """Check if cluster has a capability.
+
+    :param capability: capability name
+    :type capability: string
+    :return: does the cluster has capability
+    :rtype: bool
+    """
+    headers = {
+        'Accept': ('application/vnd.dcos.capabilities+json;charset=utf-8;'
+                   'version=v1')
+    }
+    dcos_url = config.get_config_val('core.dcos_url')
```

Comment:
we need to guard against errors. we should move `dcos.cosmospackage.enabled` here.